src/assembler/parser/index.ts

```diff
@@ -12,6 +12,8 @@ const INSTRUCTION = `([a-z]+)${SPACE_OR_TAB}+(${OPTION}(?:${SPACE_OR_TAB}*,${SPA
 const LITERAL_SYMBOL_DECLARATION = `(${SYMBOL})${SPACE_OR_TAB}+EQU${SPACE_OR_TAB}+(${VALUE})`
 const AREA_DECLARATION = `AREA${SPACE_OR_TAB}+(${SYMBOL})${SPACE_OR_TAB}*,${SPACE_OR_TAB}*(DATA|CODE)${SPACE_OR_TAB}*,${SPACE_OR_TAB}*(READ(WRITE|ONLY))`
 const COMMENT = `;[^\\n]*`
+const SPACE_OR_FILL = `SPACE|FILL|\\%`
+const SPACE_OR_FILL_INSTRUCTION = `${SPACE_OR_TAB}*(${SPACE_OR_FILL})${SPACE_OR_TAB}+([0-9]+\\*[0-9]+)`
```

Comment:
addition not possible. Only multiplication?

Reply:
![grafik](https://user-images.githubusercontent.com/56474048/169228166-a025c635-0e2e-41d1-a2e9-f3efa01dbff8.png)
additionally allow spacing between value and operation

Reply:
Please add as discussed test for + and operator within brackets

Reply:
Added this + tests.